compatibility_server/deployment/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: compatibility-server
```

Comment:
Can you change the name? Like `compatibility-server-service`

Reply:
done